src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
I take that first part back, what I'm suggesting is exactly what the Example does (just didn't see that).  It has a publicationDate property which is set to the raw JSON form of a date.  I still think the binding to native types is preferrable and am working on that alternative now.